Source/DafnyLanguageServer.Test/DafnyLanguageServer.Test.csproj

```diff
@@ -39,6 +39,12 @@
     <None Update="Lookup\TestFiles\foreign-verify.dfy">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="Lookup\TestFiles\find-refs-a.dfy">
```

Comment:
Nit: I think it's easier to read tests if all the relevant code is in the test. Alternatives to this are either opening these files in memory, or writing them to disk in the test.